test/util/test_geojson.py

```diff
@@ -0,0 +1,46 @@
+import unittest
```

Comment:
License is missing.

Reply:
License? Which license, where?

Reply:
The MIT license, just as geom.py, ts.py or ancvar.py have it. I just noticed that I left this comment in the wrong place. It was meant to be placed at the beginning of geojson.py.